core/src/test/java/nl/nn/adapterframework/xml/ClassLoaderXmlEntityResolverTest.java

```diff
@@ -1,198 +0,0 @@
-package nl.nn.adapterframework.xml;
-
-import static org.hamcrest.CoreMatchers.startsWith;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThrows;
-
-import java.io.IOException;
-import java.net.URL;
-import java.util.jar.JarFile;
-
-import org.apache.xerces.xni.XMLResourceIdentifier;
-import org.apache.xerces.xni.XNIException;
-import org.apache.xerces.xni.parser.XMLInputSource;
-import org.junit.Ignore;
-import org.junit.Test;
-import org.xml.sax.SAXException;
-
-import nl.nn.adapterframework.configuration.classloaders.JarFileClassLoader;
-import nl.nn.adapterframework.core.IScopeProvider;
-import nl.nn.adapterframework.testutil.TestScopeProvider;
-
-public class ClassLoaderXmlEntityResolverTest {
```

Comment:
is het nog een idee dit gedeeltelijk naar de ClassLoaderURIResolverTest te verplaatsen?